src/Main.cpp

```diff
@@ -19,6 +20,11 @@ void printThreadExample(void* param) {
     UserInterface ui = UserInterface();
     Logger logger = Logger(LogLevel::Debug, ui);
 
+    auto socket = SocketContainer::getHostClientSocket("127.0.0.1", 5555, logger);
```

Comment:
Similarly to the PR for the robot counterpart, the port for the socket should be supplied by the launch file. I would add it now to simplify the integration in the future. Also, the container functions should not take any parameters in my opinion. The container function should fetch all dependencies and configuration necessary.

Reply:
Fixed, inject using ros node handle, idk how to add those args though, we don't have a launch file here, it's in Swarmus-ROS right?